src/sheets/v2/mixin.js

```diff
@@ -522,6 +522,24 @@ const WarhammerSheetMixinV2 = (cls) => class extends cls
             })});
         }
     }
+
+    _onFirstRender(context, options) {
+        super._onFirstRender(context, options);
+        const containers = {};
+        for (const [part, config] of Object.entries(this.constructor.PARTS)) {
+            if (!config.container?.id) continue;
+            const element = this.element.querySelector(`[data-application-part="${part}"]`);
+            if (!element) continue;
+            if (!containers[config.container.id]) {
+                const div = document.createElement("div");
+                div.dataset.containerId = config.container.id;
+                div.classList.add(...config.container.classes ?? []);
+                containers[config.container.id] = div;
+                element.replaceWith(div);
+            }
+            containers[config.container.id].append(element);
```

Comment:
Not sure what this is accomplishing?

Reply:
Wraps some PARTS in common "container" element without need to make single PART with partials
![image](https://github.com/user-attachments/assets/e115a461-e8a4-4c3c-8865-d5fb1a2e4016)
![image](https://github.com/user-attachments/assets/cee5592e-e026-493f-a781-ff6505e14d9e)